src/js/overlay.js

```diff
@@ -9,11 +9,6 @@ const checkOptions = function(opts) {
 		opts.trigger = document.querySelector(opts.trigger);
 	}
 
-	// There can't be a heading with an empty title
```

Comment:
Oh actually this check should still be here because the title is still used for the aria label